xwiki-pro-macros-confluence-bridges/xwiki-pro-macros-confluence-bridges-api/src/main/java/com/xwiki/macros/confluence/detailssummary/macro/DetailsSummaryMacroParameters.java

```diff
@@ -0,0 +1,293 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package com.xwiki.macros.confluence.detailssummary.macro;
+
+/**
+ * Parameter bean class for the DetailsSummaryMacro. This class holds all the parameters that can be configured for the
+ * macro.
+ * @version $Id$
+ * @since 1.28.3
+ */
+public class DetailsSummaryMacroParameters
+{
+    private String id = "";
+
+    private String headings = "";
+
+    private String cql = "";
+
+    private String label = "";
+
+    private String firstcolumn = "";
+
+    private boolean showLastModified;
+
+    private boolean showPageLabels;
+
+    private boolean showCreator;
+
+    private String operator = "OR";
+
+    private int max = 1000;
+
+    private String sort = "";
+
+    private boolean reverse;
+
+    /**
+     * Returns the unique identifier for the macro instance.
+     *
+     * @return the id
```

Comment:
```suggestion
     * @return the unique identifier for the macro instance
```

More generally in this whole class you can do it like this to avoid repetitions

For instance, setters don't need a description, the `@param` line suffices I think.